src/containers/App/index.test.js

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
 import { mount } from 'enzyme';
-import { render, cleanup } from '@testing-library/react';
+import { render, cleanup, act } from '@testing-library/react';
 import { withAppContext } from 'test/utils';
+import { createMemoryHistory } from 'history';
 import App, { AppContainer, mapDispatchToProps } from './index';
 import { REQUEST_CATEGORIES } from './constants';
 
 jest.mock('components/MapInteractive');
 
+jest.useFakeTimers();
+
 describe('<App />', () => {
+  let listenSpy;
+  let spyScrollTo;
+  const history = createMemoryHistory();
```

Comment:
you can use the `history` object provided by the test utils

Reply:
die zit niet in deze branch  maar wel gedaan